staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -196,7 +196,7 @@ func conditionFuncFor(condition string, errOut io.Writer) (ConditionFunc, error)
 		return IsCreated, nil
 
 	case strings.HasPrefix(lowercaseCond, "condition="):
```

Comment:
I would suggest remaining strict that condition= and jsonpath= must be case-sensitive, as they were before https://github.com/kubernetes/kubernetes/pull/125868/files

```
strings.HasPrefix(condition, "condition=") {

strings.HasPrefix(condition, "jsonpath=") {
```

Otherwise we will get people accidentally scripting `--for=Condition=...` or `--for=JSONPath=...` and those scripts will fail on old kubectl

Reply:
Yeah, I guess that's fair assumption.